### PR TITLE
fixes #11308 - don't use deprecated proxy scope

### PR DESCRIPTION
**Pull request by mmoll**
Maybe merge together with #24 and bump the version to 1.0.0 because of Foreman version dependencies.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 # Global parameters used in configuration snippets
 parameters = [
-  { :name => 'chef_handler_foreman_url', :value => SmartProxy.chef_proxies.first.try(:url) || Setting.foreman_url },
+  { :name => 'chef_handler_foreman_url', :value => SmartProxy.with_features('chef').first.try(:url) || Setting.foreman_url },
```
Comment (ares) [inline db/seeds.rb]: capital `C` in `Chef` is needed

**Comment by ares** [comment]
This change will be compatible with Foreman 1.5+, we just have to release new version which will make it to 1.9 repo. The linked PR will need the bump for sure but I was thinking about 0.2, 1.0.0 would sound too stable.

File: db/seeds.rb (hunk 1 above)
Comment (mmoll) [inline db/seeds.rb]: done.

**Comment by ares** [comment]
Thanks @mmoll, merging!

**Pull request merged**